packages/labs/ssr/src/lib/dom-shim.ts

```diff
@@ -60,8 +64,12 @@ export const getWindow = ({
     hasAttribute(name: string) {
       return attributesForElement(this).has(name);
     }
-    attachShadow() {
-      return {host: this};
+    attachShadow(init?: ShadowRootInit) {
```

Comment:
I've added this as an optional param because I believe this would technically be a breaking change if it were required, even though the platform requires the init argument. I'm happy to update this to align with the platform if that is preferred.

Reply:
I think we should probably just make it required, to match the platform. It seems like it would be unusual to be call this method _only_ in an SSR context -- it will usually be called by a library that is already web compatible. So I think we can make it required without issuing a breaking change.

Reply:
Done. I'll leave this thread open so it's easier to find for other reviewers, in case they have thoughts.